.jenkins/pytorch/win-test.sh

```diff
@@ -74,89 +75,197 @@ set PYTHONPATH=%TMP_DIR_WIN%\\build;%PYTHONPATH%
 
 if NOT "%BUILD_ENVIRONMENT%"=="" (
     cd %TMP_DIR_WIN%\\build
+    echo "about to run Python downloader script..."
     python %TMP_DIR_WIN%\\ci_scripts\\download_image.py %TMP_DIR_WIN%\\%IMAGE_COMMIT_TAG%.7z
+
+
+    echo "about to unzip archive..."
     :: 7z: -aos skips if exists because this .bat can be called multiple times
     7z x %TMP_DIR_WIN%\\%IMAGE_COMMIT_TAG%.7z -aos
+    echo "unzipped archive."
     cd %WORKING_DIR%
 ) else (
     xcopy /s %CONDA_PARENT_DIR%\\Miniconda3\\Lib\\site-packages\\torch %TMP_DIR_WIN%\\build\\torch\\
 )
 
+echo "Now at the bottom of the script."
 EOL
 
 cat >$TMP_DIR/ci_scripts/test_python_nn.bat <<EOL
+
+
+echo "KARL: runnin test_python_nn.bat"
+
 call %TMP_DIR%/ci_scripts/setup_pytorch_env.bat
+
+
 :: Some smoke tests
+echo "KARL: A1"
 cd test/
+
 :: Checking that caffe2.python is available
+echo "KARL: A2"
 python -c "from caffe2.python import core"
+
+echo "KARL: A3"
+
 if ERRORLEVEL 1 exit /b 1
+
+echo "KARL: A4"
+
+
+:: Checking that torch is importable
+python -c "import torch"
+
```

Comment:
nit: need one `if ERRORLEVEL 1 exit /b 1` here, otherwise the clause won't cause the whole script to fail.